tokio/src/task/join_set.rs

```diff
@@ -23,7 +23,10 @@ use crate::util::IdleNotifiedSet;
 ///
 /// All of the tasks must have the same return type `T`.
 ///
-/// When the `JoinSet` is dropped, all tasks in the `JoinSet` are immediately aborted.
+/// When the `JoinSet` is dropped, all *async* tasks in the `JoinSet` are
+/// immediately aborted. Tasks spawned with [`spawn_blocking`] or
+/// [`spawn_blocking_on`] can only be aborted before they start running.
+/// This is because they are not *async*, see [task cancellation].
```

Comment:
Why is "async" italic every time? We don't usually do that.

Reply:
I wanted to highlight the difference between the async tasks and the blocking ones. Its just a formatting thing, if it conflicts with the usual docs style it can be removed.